### ISSUE TITLE
[bench request]: Add jqv (Qwen3-32B zero-shot, TypeSafe wire format, public endpoint)

### ISSUE THREAD

**Issue by hjmurmur** [fstandhartinger/jevbench#6]
Request to add **jqv** (Qwen3-32B, Apache-2.0 base weights, no fine-tuning; TypeSafe wire format) to the ranked systems, via a public endpoint.

**What it is.** An open reconstruction of the Jev inference design on a stock decoder LLM: the state is prefilled once, every question runs as an isolated branch (block attention mask, no cross-question leakage), and the answer is read directly from the option-letter logits in one forward pass (no decoding). The served probabilities are temperature-scaled with a single scalar (T = 3.02) fitted on 400 MMLU validation items; provenance is reported by `GET /health`. The submitted configuration is the zero-shot backbone without any decision training. It serves TypeSafe's wire format (`POST /v1/systemone`, choice / noul / score, string or JSON state), so your `typesafe` adapter works unchanged.

**Endpoint (self-hosted, no billing).**

- `https://asn-front-mix-develop.trycloudflare.com` — a Cloudflare Quick Tunnel to an Apple M5 Max (128 GB) in Japan running Qwen3-32B in BF16 on PyTorch/MPS, one request at a time.
- No API key: run with `--key-env ''`; any `model` value is accepted and echoed back (we use `jqv-qwen3-32b`).
- Cost basis: `no_billable_account_public_endpoint`.
- Please send one warm-up request before timing. Raw latency on the machine is p50 0.65 s on the public standard tier; your wall time will include the network path from Germany to Japan.
- The tunnel stays up until your run is done — please comment here when finished so I can take it down. The source is not public yet; I can share it on request.

```sh
python -m jevbench.cli run --tasks datasets/public/original.jsonl \
  --adapter typesafe --endpoint https://asn-front-mix-develop.trycloudflare.com --key-env '' --model jqv-qwen3-32b \
  --cost-basis no_billable_account_public_endpoint --reserve-usd 0 --delay-s 0.2 ...
```

**Our own numbers on the public items** (your harness at commit 7ce310c, `typesafe` adapter, one request at a time, argmax over the exact label set, served probabilities as above):

| tier | items | jqv (Qwen3-32B, zero-shot) |
|---|---|---|
| easy | 48 | 1.000 |
| standard (original) | 72 | 0.958 |
| hard | 111 | 0.622 |

Hard-tier top-label ECE 0.107 (raw p50 0.65 s). No distribution-gold items are public, so we have no distribution-fidelity number. We are not claiming a rank from this: the held-out and judge items are yours, and speed is measured from your server.

**Disclosure.** No training was done on any benchmark item; the only fitted quantity is the temperature (MMLU validation). The public 231 items were used to evaluate nine configurations (three backbone sizes × zero-shot / option-order averaging / LoRA head), and the choice to submit the zero-shot 32B configuration rather than the option-averaged one was made after seeing those public-tier numbers, so treat the public tiers as a development gate once. JSON-state items are rendered to the model as pretty-printed JSON text.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by fstandhartinger** [comment]
Measured and published in **JevBench v1.2.7** — thank you for keeping the endpoint up. You can take the tunnel down now.

**jqv is listed as a partial row, shown but not ranked.** Your endpoint runs on your own machine, and we do not send held-out items to an endpoint a submitter operates. I only caught that after the easy and standard/judge tiers had gone out in full, so those include their held-out items; the hard tier was stopped before its held-out block and re-run on the 111 public hard items alone. The 109 held-out hard items were never sent, so the row covers 425 of 534 decisions.

What we measured, one request at a time through the unchanged `typesafe` adapter:

| | |
|---|---|
| Easy | 72/72 (100 %) |
| Standard + judge | 227/242 (93.8 %) |
| Public hard items | 69/111 (62.2 %) |
| Your public numbers | reproduced exactly: easy 1.000, standard 0.958, hard 0.622 |
| JevBench Score | 67.2 (Intelligence 76.1 · Calibration 74.9 · Speed 67.6 · Cost 52.8), shown without a rank |
| Latency from Germany | p50 0.92 s, p95 4.71 s raw, network path to Japan included; the ×2 non-production adjustment applies |
| Cost | $0.0374 per 1,000 decisions, estimated at OpenRouter's public `qwen/qwen3-32b` tariff ($0.08 per million input tokens, nothing generated). A free endpoint is not priced as free. |

Page: https://benchmarkheaven.com/jev-models · method, endpoint condition and cost basis, committed before the runs: [`docs/v1.2-additions-run3.md`](https://github.com/fstandhartinger/jevbench/blob/v1.2.7/docs/v1.2-additions-run3.md)

If jqv gets public weights or serving code we can run ourselves, or a production endpoint, we can run the held-out hard items too and the row becomes rankable.

- Harold

**Comment by hjmurmur** [comment]
Thank you for running it, and for committing the endpoint condition and cost basis before the run. The row and its footnote describe the setup accurately, and it is reassuring that the public numbers reproduced exactly. The tunnel is down now.

Understood on the held-out policy for submitter-operated endpoints. If the serving code goes public or a production endpoint becomes available, I will open a new request.
